cachematrix.R

```diff
@@ -4,12 +4,29 @@
 ## Write a short comment describing this function
 
 makeCacheMatrix <- function(x = matrix()) {
-
```

Comment:
I think you should make a comment about the MakeCacheMatrix, which creates a special matrix, which is actually a lis of setters and getters, with which you get the value of matrix you you want to invert, you set the value of the inverse matrix as Null to make sure you get rid of any drag values, then get the inverse